MekHQ/src/mekhq/campaign/parts/Part.java

```diff
@@ -184,6 +184,8 @@
     //this tracks whether the part is reserved for a refit
     protected UUID refitId;
     protected UUID reserveId;
+    //temporarily mark the part used by current refit planning
+    protected boolean usedForRefitPlanning;
```

Comment:
This variable is not added to the `writeToXML` or `generateInstanceFromXML` methods in `Part.java`. Thus it will revert to default if the game is saved and then re-loaded.

Reply:
Its possible that it is not needed to be saved, but its not clear to me from the code changes whether that is true or not.

Reply:
The value is transient